packages/react-events/src/Press.js

```diff
@@ -659,18 +704,23 @@ const PressResponder = {
           // data around for handling of the context menu
           state.pointerType = pointerType;
           state.pressTarget = context.getEventCurrentTarget(event);
+          if (isPointerEvent) {
+            state.activePointerId = pointerId;
+          } else if (isTouchEvent) {
+            const touchEvent = getTouchFromPressEvent(nativeEvent);
+            state.activePointerId = touchEvent.identifier;
```

Comment:
FYI, might be worth looking into which versions of Safari are still reusing touch identifiers - https://github.com/necolas/react-native-web/blob/36dacb2052efdab2a28655773dc76934157d9134/packages/react-native-web/src/modules/normalizeNativeEvent/index.js#L15-L22

Reply:
I'm not sure I fully follow it, is this a well known bug? It seems huge if the identifiers can be the same for events that aren't the same?!

Reply:
Safe it ignore. I can't find anything about it online or in the original commit (>3 years ago). In fact, it sounds like Safari always produces unique identifiers whereas Chrome will recycle them (but presumably in a safe way).